.gitignore

```diff
@@ -3,6 +3,7 @@
 
 ### Composer ###
 composer.phar
+composer.lock
```

Comment:
The library should support multiple PHP versions to handle different versions of moneyphp. Locking the version will restrict tests to the locked version, but we need to test with the newer version for PHP 8 and above.

Note: Clients using the library won't use the lock file, so including or excluding it will yield the same result for users.

Reply:
Okies